src/Legacy/Drupal7.php

```diff
@@ -9,9 +9,260 @@
 
 /**
  * Defines the Drupal 7 legacy service.
+ *
+ * @method void flood_register_event(string $name, integer $window, string $identifier)
+ * @method void flood_clear_event(string $name, string $identifier)
+ * @method bool flood_is_allowed(string $name, integer $threshold, integer $window, string $identifier)
+ * @method bool module_load_all(bool $bootstrap)
```

Comment:
bool $bootstrap = FALSE

etc.

So need to add all the things that are optional here.